### DISCUSSION TITLE
[Jira] use tag to identify incidents

### DISCUSSION THREAD

**Discussion by rbraeunlich** [apache/devlake#6676] [Ideas]
Hey everyone,
when configuring a Jira scope config I can only identify incidents based on the ticket type. Currently, my team is using bugs with a certain tag to identify incidents. Is there any plan to add some kind of additional tag filter to the scope config?
Cheers,
Ronny

**Comment by pogopaule** [comment]
Similar problem here. Our incidents are bugs which have the highest level.

**Comment by abeizn** [comment]
@klesh @Startrekzky

**Comment by Startrekzky** [comment]
Hi @rbraeunlich , 
1. What specific tag(s) do you use to identify incidents? 
2. Do different teams in your org use the same definition (tag) to identify incidents in Jira?

In short, I think it's a valid feature request, we just need to figure out the solutions to it. Cc. @abeizn

**Reply by rbraeunlich** [reply]
Hi @Startrekzky,
as far as I know every team has it's own rules. But mapping this to DevLake it would be the same rule per Scope Config. Since tags are free to define (at least for us), the writing may be "incident", "Incident" or something similar. There is nothing explicitly defined.

**Comment by pogopaule** [comment]
As a workaround I'm currently testing the following workflow:

1. Create two filters in Jira
  A: a filter that matches all issues that shall become DevLake Incidents. In my case these are issues with the type Bug and the priority Highest.
  B: a filter that matches every ticket you want DevLake to know of, except for those matched by filter A
2. Create a jira board X for filter A and a jira board Y for filter B
3. In DevLake add these two boards and create different Scopes for them. X's Scope only maps Incidents to Bugs and Y's Scope maps the remaining issue types.

So far this looks like a solution that could work for me. The beauty of this solution is that I can make use of jira's filter system, which does not have to be reimplemented in DevLake.

**Comment by Startrekzky** [comment]
Bump: https://devlake-io.slack.com/archives/C03APJ20VM4/p1717616557358419?thread_ts=1717408612.609139&cid=C03APJ20VM4
